src/PythonMigrationViewExtension/PythonMigrationAssistantViewModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Dynamo.PythonMigration.MigrationAssistant;
+using PythonNodeModels;
+using System.Collections.Generic;
+
+namespace Dynamo.PythonMigration
+{
+    internal class PythonMigrationAssistantViewModel
+    {
+        public string OldCode { get; set; }
+        public string NewCode { get; set; }
+
+        public PythonMigrationAssistantViewModel(PythonNode pythonNode)
+        {
+            OldCode = pythonNode.Script;
+            MigrateCode();
```

Comment:
Unsure how long does it take to migrate a very large script, but calling this on ctor might slow down window opening time.
Maybe make `MigrateCode()` public and call it on migration assistant window `Opened` event?

Reply:
only thing i'd say different would be to call it before the window opens, otherwise we have to add a `Migrating code...` state to handle potentially long times before anything is available to be displayed.

Reply:
just realised if we call it before, it's same as is now 😂 . might be overkill to add the loading/migrating state since it's be super-fast on most cases and might be visible only for a split second.